### PR TITLE
feat: CheckableSelectField - Custom MultiSelectField

### PR DESCRIPTION
**Pull request by Sodik**

### CODE REVIEW

File: packages/ui/__stories__/CheckableSelectField.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+}
+
+export const OpenWithInputValue = Template.bind({})
+OpenWithInputValue.args = {
+  ...Empty.args,
+  ...Open.args,
+  inputValue: 'obi',
+}
+
+export const OpenWithInputValueAndNoResults = Template.bind({})
```
Comment (fantkolja) [inline packages/ui/__stories__/CheckableSelectField.stories.tsx:104]: When there is an input value, some options are filtered out and I click 'Select All' also the hidden options are selected. This doesn't feel right
image

File: packages/ui/src/Select/CheckableSelectField.module.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
+@use './styles/constants' as c;
```
Comment (fantkolja) [inline packages/ui/src/Select/CheckableSelectField.module.scss:1]: Seems like the checkbox is higher than the text
image

**Review by fantkolja — CHANGES_REQUESTED** [review]
Well done!
Few comments...

**Review by utkukaratas — APPROVED** [review]
overlaps a lot with MultiSelectField :/

**Comment by Sodik** [comment]
I think we should review selects and perhaps remove react-select or refactor this component to make it more compatible with react-select way @pawelklasa

File: packages/ui/__stories__/CheckableSelectField.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react'
```
Comment (fantkolja) [inline packages/ui/__stories__/CheckableSelectField.stories.tsx:1]: nit: Probably we should wrap long options, wdut?
image

**Review by fantkolja — APPROVED** [review]

**Pull request merged**